io.gpt4all.gpt4all.yml

```diff
@@ -0,0 +1,136 @@
+app-id: io.gpt4all.gpt4all
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '6.6'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+  - --env=LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app/lib/x86_64-linux-gnu/
```

Comment:
this won't work as variable substitutions aren't done. Also I'm sure it's unecessary. Also what does this do on `aarch64`?

Reply:
The problem is that the runtime linker does not find `libQt6HttpServer.so.6`, when it is not in the linker path

Reply:
if you mean "at build time" this is even more wrong.

Reply:
Not at build time. When I try to start the application.

Reply:
It should be installed to `/app/lib`, if not you can add a symlink there.